packages/@react-native-windows/tester/src/js/examples-win/Text/TextExample.windows.tsx

```diff
@@ -197,9 +197,11 @@ export class TextExample extends React.Component<
               </Text>
             </Text>
             <Text onPress={() => this.setState({toggle2: !this.state.toggle2})}>
-              Click to change raw text:{' '}
-              <Text style={{textTransform: 'uppercase'}}>
-                Hello {this.state.toggle2 ? 'Earth' : 'World'}
+              <Text>
```

Comment:
Why do you need this new nested Text tag here?

Reply:
We don't, adding it allowed me to create the repro above. It doesn't hurt to keep it there, but we can revert it if you'd like.

Reply:
I was thinking more of keeping both scenarios - if the new one helped catch this edge case, it looks likely the old one might do that as well in the future.

Reply:
Unless that's covered in another example already.

Reply:
There should be more than enough coverage from the other examples.